src/plugins/lua/bindings/lua_Constants.cpp

```diff
@@ -0,0 +1,283 @@
+         "MonsterSummoning",           "Monster Summoning",
+         "InvisibleMan",               "Invisible Man",
+         "Special",                    "Special"
+      );
+
+      // Spawn Consider Colors
+      lua.new_enum("ConsiderColors",
+         "Grey",                       "GREY",
+         "Green",                      "GREEN",
+         "LightBlue",                  "LIGHT BLUE",
+         "Blue",                       "BLUE",
+         "White",                      "WHITE",
+         "Yellow",                     "YELLOW",
+         "Red",                        "RED"
+      );
+
+      // Spawn Diety Names
+      lua.new_enum("DietyNames",
+         "Agnostic",                  "Agnostic",
+         "Bertoxxulous",              "Bertoxxulous",
+         "BrellSerilis",              "Brell Serilis",
+         "CazicThule",                "Cazic Thule",
+         "ErollsiMarr",               "Erollsi Marr",
+         "Bristlebane",               "Bristlebane",
+         "Innoruuk",                  "Innoruuk",
+         "Karana",                    "Karana",
+         "MithanielMarr",             "Mithaniel Marr",
+         "Prexus",                    "Prexus",
+         "Quellious",                 "Quellious",
+         "RallosZek",                 "Rallos Zek",
+         "RodcetNife",                "RodcetNife",
+         "SolusekRo",                 "Solusek Ro",
+         "TheTribunal",               "The Tribunal",
+         "Tunare",                    "Tunare", 
+         "Veeshan",                   "Veeshan"
+      );
+      
+      // Spawn Equipment Slot Names
+      lua.new_enum("EquipmentSlots",
+         "Head",                      "head",
+         "Chest",                     "chest",
+         "Arms",                      "arms",
+         "Wrists",                    "wrists",
+         "Hands",                     "hands",
+         "Legs",                      "legs",
+         "Feet",                      "feet",
+         "Primary",                   "primary",
+         "Offhand",                   "offhand"
+      );
+
+      // Spawn Gender
+      lua.new_enum("Gender",
+         "Male",                      "MALE",
+         "Female",                    "FEMALE"
+      );
+
+      // Spawn Invis Types
+      lua.new_enum("InvisTypes",
+         "Any",                       0,
+         "Normal",                    1,
+         "Undead",                    2,
+         "Animal",                    3,
+         "SOS",                       4
+      );
+
+      // Spawn Player States
+      lua.new_enum("PlayerStates",
+         "Idle",                      0,
+         "Open",                      1,
+         "WeaponSheathed",            2,
+         "Aggressive",                4,
+         "ForcedAggressive",          8,
+         "InstrumentEquipped",        0x10,
+         "Stunned",                   0x20,
+         "PrimaryWeaponEquipped",     0x40,
+         "SecondaryWeaponEquipped",   0x80
```

Comment:
Where would these value get used?

Also, if we're going to add these values they need to be bound to actual constants instead of magic numbers.

Reply:
This was an interesting one when I wrote it.  The spawn object has a member, `PlayerState` that "returns a mask as an inttype which has the following meaning: 0=Idle 1=Open 2=WeaponSheathed 4=Aggressive 8=ForcedAggressive 0x10=InstrumentEquipped 0x20=Stunned 0x40=PrimaryWeaponEquipped 0x80=SecondaryWeaponEquipped"

So my assumption on it's use in Lua would be an easier way to check the state, such as

```lua
local mySpawn = mq.TLO.Me.Target()
if mySpawn.PlayerState() == Constants.PlayerState.SecondaryWeaponEquipped then
-- Do Stuff
end
```

Reply:
I take it back, lol.  I still think this is useful.  Updated the answer of where this could be used with an example.
```lua
local mySpawn = mq.TLO.Me.Target()
if mySpawn.PlayerState() == (Constants.PlayerState.PrimaryWeaponEquipped + Constants.PlayerState.SecondaryWeaponEquipped) then
-- Do Stuff
end
```